src/main/java/io/citrine/jcc/search/core/query/DataScope.java

```diff
@@ -0,0 +1,93 @@
+package io.citrine.jcc.search.core.query;
+
+import com.fasterxml.jackson.annotation.JsonIgnore;
+import io.citrine.jcc.util.ListUtil;
+
+import java.util.List;
+
+/**
+ * Definition of a data scope.
+ * 
+ * @author Kyle Michel
+ */
+public class DataScope {
+
+    /**
+     * Set the list of queries. This replaces any filters that are already present.
+     *
+     * @param query List of {@link DataQuery} objects.
+     * @return This object.
+     */
+    public DataScope setQuery(final List<DataQuery> query) {
```

Comment:
Every query has a logic associated with it. For example, say that query had four elements (A, B, C, and D) and that they have SHOULD, SHOULD, MUST, and MUST_NOT respectively. Then they resulting query would be (A or B) and C and not D.